api/HTMLVideoElement.json

```diff
@@ -611,6 +611,57 @@
           }
         }
       },
+      "msIsStereo3D": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/msIsStereo3D",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": false
+            }, 
```

Comment:
This line has trailing whitespace (I really should probably add something to the error output to highlight this).